bpf/kmesh/include/route_config.h

```diff
@@ -105,7 +105,7 @@ virtual_host_match(Route__RouteConfiguration *route_config, address_t *addr, ctx
     void *ptrs = NULL;
     Route__VirtualHost *virt_host = NULL;
     Route__VirtualHost *virt_host_allow_any = NULL;
-    char uri_key[4] = {'U', 'R', 'I', '\0'};
+    char host_key[5] = {'H', 'o', 's', 't', '\0'};
```

Comment:
Where is `host_key` header value read

Reply:
The kmesh.ko file parses and saves the HTTP packet header. The user-defined bpf_get_msg_header_element helper function is added to read the header.